Library/ViewModels/RewardCardContainerViewModel.swift

```diff
@@ -78,14 +78,12 @@ public final class RewardCardContainerViewModel: RewardCardContainerViewModelTyp
       .map { $0.id }
 
     // Tracking
-    projectAndRewardOrBacking
+    projectAndReward
       .takeWhen(self.pledgeButtonTappedProperty.signal)
-      .observeValues { projectAndRewardOrBacking in
```

Comment:
I think we should actually refactor this view model to only accept a `reward` for configuration -- right now it accepts an `Either<Reward, Backing>` type, although we never actually use the `Backing` configuration.